src/container/srv_target.c

```diff
@@ -134,6 +208,9 @@ cont_stop_agg_ult(struct ds_cont_child *cont)
 {
 	int rc;
 
+	if (!cont->sc_vos_aggregating)
```

Comment:
I don't know why this extra checking is required, but it could cause 'sc_agg_ult' leaking when the aggregation ULT exited before calling this cont_stop_agg_ult() function.

So you'd remove this check or move it after the 'sc_agg_ult' free below.

Reply:
OK, I will fix it in next commit.